src/tests/functional/plugin/conformance/test_runner/api_conformance_runner/src/ov_plugin/properties.cpp

```diff
@@ -6,6 +6,11 @@
 #include "base/ov_behavior_test_utils.hpp"
 #include "openvino/runtime/properties.hpp"
 #include "ov_api_conformance_helpers.hpp"
+#define ADD_MANDATORY_PREFIX(GET_TEST_NAME)                                                           \
```

Comment:
We can rename the macros to `MARK_MANDATORY_FOR_HW_DEVICE` and place to the common place as `src/tests/functional/plugin/shared/include/base/ov_behavior_test_utils.hpp`